bin/node/src/commands/mod.rs

```diff
@@ -78,4 +79,12 @@ pub struct BlockProducerConfig {
     /// in-process which is expensive.
     #[arg(long = "block-prover.url", env = ENV_BLOCK_PROVER_URL, value_name = "URL")]
     pub block_prover_url: Option<Url>,
+
+    /// The number of transactions per batch.
+    #[arg(long = "max-txs-per-batch", default_value_t = SERVER_MAX_TXS_PER_BATCH)]
+    max_txs_per_batch: usize,
+
+    /// Maximum number of batches per block.
+    #[arg(long = "max-batches-per-block", default_value_t = SERVER_MAX_BATCHES_PER_BLOCK)]
+    max_batches_per_block: usize,
```

Comment:
Do we not want these to be set via the env variables? If we do, shouldn't these have `env` attributes?

Reply:
Ah yeah. I've created a follow-up issue.